support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -657,10 +657,14 @@ function CheckoutComponent({ geoId }: Props) {
 			if (processPaymentResponse.status === 'success') {
 				const order = {
 					firstName: personalData.firstName,
+					email: personalData.email,
 					price: price,
 					product: productId,
 					ratePlan: query.ratePlan,
 					paymentMethod: paymentMethod as string,
+					// TODO - get this from the /identity/get-user-type endpoint (new/guest/existing)
+					userTypeFromIdentityResponse: 'new',
```

Comment:
This isn't necessarily related to the order information.

We could avoid adding this to the order info and determine the value on the thank you page from on `isSignedIn ? 'new' : 'current'`. 

If we want, for now, could we add it to the thank you page rather than hijack this functionality?

We could do this as part of another PR (but let's create an issue).

Reply:
Can see its not worth putting in session storage a scould be gained from elsewhere in future (seperate PR). Removed it from here - updated.